packages/clerk-js/src/utils/email.ts

```diff
@@ -4,6 +4,6 @@ export type BuildEmailAddressParams = {
 };
 
 export function buildEmailAddress({ localPart, frontendApi }: BuildEmailAddressParams): string {
-  const domain = frontendApi ? frontendApi.replace('clerk.', '') : 'clerk.dev';
+  const domain = frontendApi ? frontendApi.replace('clerk.', '') : 'clerk.com';
```

Comment:
@dimkl our website still mentions `support@clerk.dev`, let's make sure we support the `.com` and if we do let's create a ticket in order to update it for the marketing as well.

<img width="202" alt="image" src="https://github.com/clerkinc/javascript/assets/19269911/8123b5d8-4324-468e-adcd-7cc5b3084db1">

Reply:
Based on the implementation and usage of `buildEmailAddress()` we use the frontendApi to generate the email domain. This change probably won't affect the support email address.
I will test it more before merging this.